src/tests/electron/common/view-controllers/app-controller.ts

```diff
@@ -125,4 +127,22 @@ export class AppController {
             },
         );
     }
+
+    public getServerLog(logName: string, extraLogNames: string): string {
+        return fs
+            .readFileSync(
+                path.normalize(
+                    `drop/mock-adb/logs/${logName}/${extraLogNames}/testLogs/server.log`,
```

Comment:
It feels odd to repurpose the mock-adb folder for server logs. Would it logical/practical instead to create a per-test log folder (independent of the mock-adb folder) and have a way to tell dump both the mock-adb and server logs into that folder?

Reply:
Per offline conversation, leaving this as is because the server is being started and given this logging location by mock adb